stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SameAsShippingElementUI.kt

```diff
@@ -17,6 +20,7 @@ fun SameAsShippingElementUI(
     val label by controller.label.collectAsState()
 
     CheckboxElementUI(
+        modifier = Modifier.padding(vertical = 4.dp),
```

Comment:
We render `SameAsShipping` really weirdly. It is the only element that uses the [shouldRenderOutsideCard](https://github.com/stripe/stripe-android/blob/master/stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SameAsShippingElement.kt#L10-L12) attribute of [SectionFieldElement](https://github.com/stripe/stripe-android/blob/master/stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SectionFieldElement.kt#L9-L12) which is then taken into account when using [SectionElementUI](https://github.com/stripe/stripe-android/blob/master/stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SectionElementUI.kt#L45-L63).

This may be due to [AddressSpec](https://github.com/stripe/stripe-android/blob/2e5750109be96532c2c542ac67b8263906d33798/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressSpec.kt#L91-L97) being the only element to technically return multiple elements but [TransformSpecToElements](https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/lpmfoundations/luxe/TransformSpecToElements.kt#L53-L82) expects every `transform` to only return a single element.

I think we can solve by having every `transform` function for all the spec types return a list of elements if needed then `flatMap` in `TransformSpecToElements`. This way we can also remove `contentOutsideOfCard` and simplify what `Card` is meant to represent.